### PR TITLE
Add docs folder

### PR DESCRIPTION
**Pull request by mkg33**
For future use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +65,7 @@ prob = ODEProblem(lorenz,u0,tspan)
 sol = solve(prob,Tsit5())
 ```
 
-For "refined ODEs" like dynamical equations and `SecondOrderODEProblem`s, refer to the [DiffEqDocs](https://docs.juliadiffeq.org/dev/types/ode_types/). For example, in [DiffEqTutorials.jl](https://github.com/SciML/DiffEqTutorials.jl) we show how to solve equations of motion using symplectic methods:
+For "refined ODEs", like dynamical equations and `SecondOrderODEProblem`s, refer to the [DiffEqDocs](https://docs.juliadiffeq.org/dev/types/ode_types/). For example, in [DiffEqTutorials.jl](https://github.com/SciML/DiffEqTutorials.jl) we show how to solve equations of motion using symplectic methods:
```
Comment (devmotion) [inline README.md:68]:
```suggestion
For "refined ODEs", like dynamical equations and `SecondOrderODEProblem`s, refer to the [DiffEqDocs](https://diffeq.sciml.ai/dev/types/ode_types/). For example, in [DiffEqTutorials.jl](https://github.com/SciML/DiffEqTutorials.jl) we show how to solve equations of motion using symplectic methods:
```

File: docs/src/usage.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Usage
+
+OrdinaryDiffEq.jl is part of the SciML common interface, but can be used independently of DifferentialEquations.jl. The only requirement is that the user passes an OrdinaryDiffEq.jl algorithm to `solve`. For example, we can solve the [ODE tutorial from the docs](https://docs.juliadiffeq.org/dev/tutorials/ode_example/) using the `Tsit5()` algorithm:
```
Comment (devmotion) [inline docs/src/usage.md:3]:
```suggestion
OrdinaryDiffEq.jl is part of the SciML common interface, but can be used independently of DifferentialEquations.jl. The only requirement is that the user passes an OrdinaryDiffEq.jl algorithm to `solve`. For example, we can solve the [ODE tutorial from the docs](https://diffeq.sciml.ai/dev/tutorials/ode_example/) using the `Tsit5()` algorithm:
```

File: docs/src/usage.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+function lorenz(u,p,t)
+ SA[10.0(u[2]-u[1]),u[1]*(28.0-u[3]) - u[2],u[1]*u[2] - (8/3)*u[3]]
+end
+u0 = SA[1.0;0.0;0.0]
+tspan = (0.0,100.0)
+prob = ODEProblem(lorenz,u0,tspan)
+sol = solve(prob,Tsit5())
+```
+
+For "refined ODEs", like dynamical equations and `SecondOrderODEProblem`s, refer to the [DiffEqDocs](https://docs.juliadiffeq.org/dev/types/ode_types/). For example, in [DiffEqTutorials.jl](https://github.com/SciML/DiffEqTutorials.jl) we show how to solve equations of motion using symplectic methods:
```
Comment (devmotion) [inline docs/src/usage.md:47]:
```suggestion
For "refined ODEs", like dynamical equations and `SecondOrderODEProblem`s, refer to the [DiffEqDocs](https://diffeq.sciml.ai/dev/types/ode_types/). For example, in [DiffEqTutorials.jl](https://github.com/SciML/DiffEqTutorials.jl) we show how to solve equations of motion using symplectic methods:
```

File: docs/src/usage.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+initial_velocities = [0.5,0.0]
+prob = SecondOrderODEProblem(HH_acceleration,initial_velocities,initial_positions,tspan)
+sol2 = solve(prob, KahanLi8(), dt=1/10);
+```
+
+Other refined forms are IMEX and semi-linear ODEs (for exponential integrators).
+
+## Available Solvers
+
+For the list of available solvers, please refer to the [DifferentialEquations.jl ODE Solvers](https://docs.juliadiffeq.org/dev/solvers/ode_solve/), [Dynamical ODE Solvers](http://docs.juliadiffeq.org/dev/solvers/dynamical_solve/), and the [Split ODE Solvers](http://docs.juliadiffeq.org/dev/solvers/split_ode_solve/) pages.
```
Comment (devmotion) [inline docs/src/usage.md:66]:
```suggestion
For the list of available solvers, please refer to the [DifferentialEquations.jl ODE Solvers](https://diffeq.sciml.ai/dev/solvers/ode_solve/), [Dynamical ODE Solvers](http://diffeq.sciml.ai/dev/solvers/dynamical_solve/), and the [Split ODE Solvers](http://diffeq.sciml.ai/dev/solvers/split_ode_solve/) pages.
```

File: README.md (hunk 1 above)
Comment (mkg33) [reply README.md:68]: Thanks for pointing this out! I just copied the README.md as it was.

Comment (devmotion) [reply README.md:68]: Yes, unfortunately some parts are still outdated and weren't updated after the transition to SciML.

Comment (mkg33) [reply README.md]: I'll also go carefully through the links in other packages and check for inconsistencies.

**Pull request merged**